babel.config.js

```diff
@@ -35,6 +35,7 @@ const defaultAlias = {
   '@material-ui/styles': './packages/material-ui-styles/src',
   '@material-ui/system': './packages/material-ui-system/src',
   '@material-ui/utils': './packages/material-ui-utils/src',
+  classnames: 'clsx',
```

Comment:
Should only use this for the docs. This adds a lot of indirection to the actual packages.